config/adhoc.yml.erb

```diff
@@ -35,3 +35,5 @@ db_credential_admin: !StackSecret
 db_credential_writer: !StackSecret
 db_credential_reader: !StackSecret
 
+# aiproxy credentials
+aiproxy_api_key: 'MRo6Vw0p2joeGDUHGPg3MB2stpT837dm'
```

Comment:
Is this ok to have here?

Reply:
nope. this needs to go in locals.yml on the adhoc. and the key needs to be changed

Reply:
you could also set it up to pull from AWS Secrets Manager, which you'll need to do for the production environment anyway.

Reply:
This is a temp solution that @cat5inthecradle and I came up with for running the adhoc. It's set up to pull from secrets in all other instances, and this key will be deleted once we're done with the adhoc. I can't ssh into the server because we didn't expose that port, but I can check with Darin about that.

Reply:
```suggestion
aiproxy_api_key: !Secret
```